cmd/openshift-install/create.go

```diff
@@ -236,21 +236,25 @@ func destroyBootstrap(ctx context.Context, directory string) (err error) {
 	)
 	if err != nil {
 		logrus.Error(errors.Wrap(err, "waiting for bootstrap-complete"))
+		if err == wait.ErrWaitTimeout {
+			return err
+		}
 		return nil
 	}
 
 	logrus.Info("Destroying the bootstrap resources...")
 	return destroybootstrap.Destroy(rootOpts.dir)
 }
 
-// logComplete prints info upon completion
+// logComplete prints info and suggestions to manage installed cluster
 func logComplete(directory string) error {
 	absDir, err := filepath.Abs(directory)
 	if err != nil {
 		return err
 	}
 	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
-	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s' to manage your cluster.", kubeconfig)
-	logrus.Info("After exporting your kubeconfig, run 'oc -h' for a list of OpenShift client commands.")
+	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s' to manage the cluster.", kubeconfig)
+	logrus.Info("Wait for workers to become ready. Run 'oc get nodes | grep worker | grep Ready' to check if workers are up.")
```

Comment:
Is this the recommended approach?  It seems like `oc get nodes -w` might be more efficient (no polling).  But it wouldn't be worker-specific.  Maybe you could address that with `--selector`?  Or maybe there are likely to be few enough masters (compared to workers) in most (all?) clusters that you don't need to bother filtering the masters out here?

Reply:
I was going to ask (you) what the most efficient way was :) but do you agree that suggesting that one wait for workers to be created is appropriate?